### DISCUSSION TITLE
How to prevent the confirmation window from popping up when reconnecting

### DISCUSSION THREAD

**Discussion by soengames** [lupidan/apple-signin-unity#160] [Q&A]
hi.

I am using this plugin for connection in Firebase.

Every time I connect, the connection confirmation window (screen to enter password or fingerprint) pops up, which is inconvenient for me.

My game also has authentication via Facebook, in which case I avoid displaying the authentication UI by saving the key that generates the credential to disk and recreating it.

Is there any solution like this? 

(All methods of saving credentials have failed. Is there another way?)

**Comment by lupidan** [comment]
I don't understand the question
